packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -6,366 +6,785 @@
-                        required
-                        aria-describedby="accountTypeDescription"
-                    ></select>
-                    <span
-                        class="oui-icon oui-icon-chevron-down"
-                        aria-hidden="true"
-                    ></span>
-                </label>
-                <span
-                    data-ng-if="$ctrl.exchangeServiceInfrastructure.isDedicatedCluster()"
-                >
-                    <span
-                        id="accountTypeDescription"
-                        class="sr-only"
-                        data-ng-bind=":: 'exchange_account_add_accountType_tooltip' | translate"
-                    ></span>
-                    <span
-                        class="oui-icon ml-4 mt-3 oui-icon-help-circle"
-                        data-oui-tooltip="{{:: 'exchange_account_add_accountType_tooltip' | translate }}"
-                        aria-hidden="true"
-                    ></span>
-                </span>
+        <!--Account info section-->
+        <div class="pl-3">
+            <div class="row">
+                <p class="oui-heading_4 p-0" data-translate="exchange_account_add_personal_account_info_title"></p>
             </div>
-        </div>
 
-        <div
-            class="form-group"
-            data-ng-class="{ 'oui-field_error has-error': $ctrl.newAccountForm.login.$invalid && $ctrl.newAccountForm.login.$touched }"
-        >
-            <label
-                class="control-label"
-                for="login"
-                data-ng-bind=":: 'exchange_account_add_emailAddress_label' | translate"
-            ></label>
-            <div class="input-group w-50">
-                <input
-                    type="text"
-                    class="form-control"
-                    id="login"
-                    maxlength="256"
-                    name="login"
-                    required
-                    data-ng-model="$ctrl.newAccount.login"
-                    data-ng-model-options="{ updateOn: 'blur' }"
-                    data-ng-pattern="/^(?:[a-z0-9]+(?:[-_][a-z0-9]+)*)(?:(?:\.|\+)(?:[a-z0-9]+(?:[-_][a-z0-9]+)*))*$/"
-                    data-ng-change="$ctrl.checkEmailAddressIsAlreadyTaken()"
-                />
-                <span class="input-group-addon">@</span>
-                <select
-                    class="form-control"
-                    data-ng-model="$ctrl.newAccount.domain"
-                    data-ng-options="domain.displayName for domain in $ctrl.accountCreationOptions.availableDomains | orderBy:'displayName' track by domain.displayName"
-                    required
-                    data-ng-change="$ctrl.checkEmailAddressIsAlreadyTaken()"
-                ></select>
+            <!--Account type-->
+            <div class="row">
+                <div class="col-sm-12 col-md-10">
+                    <div class="row">
+                        <div class="w-100 oui-field pr-2"
```

Comment:
**issue(blocking):** No need to use nested column here.
You can use following structure
```html
<div class="container">
  <oui-back-button>...</oui-back-button>
  <div class="row">
    <div class="col-sm-12 col-md-10">
      <p>...</p>
      <div class="oui-field">...</div>
      ...
    </div>
  </div>
</div>
```

This layout is valid for the whole form.
Same for following file:
packages/manager/modules/exchange/src/account/update/account-update.html

Reply:
Fixed

Reply:
I just update the sections, now i have this.

```
div (container)
  div (account info section)
      p (title of section)
      div.row (manage fields grid display)
  
  div (company info section)
      p (title of section)
      div.row (manage fields grid display)
  
  div (contact info section)
      p (title of section)
      div.row (manage fields grid display)
```
    
Like this i preserve the consistency of each section.